app/controllers/api/current/patients_controller.rb

```diff
@@ -4,7 +4,9 @@ def sync_from_user
   end
 
   def sync_to_user
-    __sync_to_user__('patients')
+    unscope_associations do
+      __sync_to_user__('patients')
+    end
```

Comment:
Yes, we've introduced a default `kept` scope that excludes discarded patients. However, we still want to sync these patients and their records, so that the app is aware and can cleanup these records on the client side.